tests/mock/Concrete/PlatformOrderDecorator.php

```diff
@@ -0,0 +1,899 @@
+            'billet',
+            $additionalInformation
+        );
+
+        $paymentData[$boletoDataIndex][] = $newPaymentData;
+    }
+
+    private function extractPaymentDataFromMundipaggBillet(
+        $additionalInformation,
+        &$paymentData,
+        $payment
+    )
+    {
+        $moneyService = new MoneyService();
+        $newPaymentData = new \stdClass();
+        $newPaymentData->amount =
+            $moneyService->floatToCents($this->platformOrder->getGrandTotal());
+
+        $boletoDataIndex = BoletoPayment::getBaseCode();
+        if (!isset($paymentData[$boletoDataIndex])) {
+            $paymentData[$boletoDataIndex] = [];
+        }
+
+        if ($additionalInformation['billet_buyer_checkbox']) {
+            $newPaymentData->customer = $this->extractMultibuyerData(
+                'billet',
+                $additionalInformation
+            );
+        }
+
+        $paymentData[$boletoDataIndex][] = $newPaymentData;
+    }
+
+    public function getShipping()
+    {
+        $moneyService = new MoneyService();
+        /** @var Shipping $shipping */
+        $shipping = null;
+        $quote = $this->getQuote();
+        /** @var \Magento\Quote\Model\Quote\Address $platformShipping */
+        $platformShipping = $quote->getShippingAddress();
+
+        $shippingMethod = $platformShipping->getShippingMethod();
+        if ($shippingMethod === null) { //this is a order without a shipping.
+            return null;
+        }
+
+        $shipping = new Shipping();
+
+        $shipping->setAmount(
+            $moneyService->floatToCents($platformShipping->getShippingAmount())
+        );
+        $shipping->setDescription($platformShipping->getShippingDescription());
+        $shipping->setRecipientName($platformShipping->getName());
+
+        $telephone = $platformShipping->getTelephone();
+        $phone = new Phone($telephone);
+
+        $shipping->setRecipientPhone($phone);
+
+        $address = $this->getAddress($platformShipping);
+        $shipping->setAddress($address);
+
+        return $shipping;
+    }
+
+    protected function getAddress($platformAddress)
+    {
+        $address = new Address();
+        $addressAttributes =
+            MPSetup::getModuleConfiguration()->getAddressAttributes();
+
+        $addressAttributes = json_decode(json_encode($addressAttributes), true);
+        $allStreetLines = $platformAddress->getStreet();
+
+        $this->validateAddress($allStreetLines);
+        $this->validateAddressConfiguration($addressAttributes);
+
+        if (count($allStreetLines) < 4) {
+            $addressAttributes['neighborhood'] = "street_3";
+            $addressAttributes['complement'] = "street_4";
+        }
+
+        foreach ($addressAttributes as $attribute => $value) {
+            $value = $value === null ? 1 : $value;
+
+            $street = explode("_", $value);
+            if (count($street) > 1) {
+                $value = intval($street[1]) - 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function intval() is discouraged; use (int) construction instead.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)